ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/GLTextureStruct.cpp

```diff
@@ -1,30 +0,0 @@
-/** Copyright (C) 2008-2013 Josh Ventura, Robert B. Colton
```

Comment:
You know how fundies wanted me to roll back those other integration testing PRs you submitted? This is an example of a great cleanup change that gets rolled back for no reason when that happens. I'm not going to ask you to undo this now, because my only point is that lumping changes in with other changes isn't worth it because they might get undone in gloss. So please just make a new PR for things like this in the future; it'll take two seconds to approve and submit.

Reply:
Sorry about that, I've undone it: 40315a3360d36b14bc9ac0242c97ef00cce6855f

I agree, but I didn't see you said not to worry about it until after undoing it. Regardless, I've already made this same change over in several of the graphics related branches we're working on.

I'll stop lumping in random changes like this, I promise, it just becomes a bit of a bad habit. Also, I wish you would have barked at fundies when I did about an analogous thing he did on #1322.

Reply:
Ok, resetting the file turned out to be a pain in the ass, so I did keep the deletion for now: c845ad7c07ba745abff1aef093ad0da7347513de